62_CAD/fragment_shader.hlsl

```diff
@@ -372,6 +372,23 @@ float32_t4 calculateFinalColor<true>(const uint2 fragCoord, const float localAlp
     return col;
 }
 
+// TODO: place this MSDF code elsewhere
```

Comment:
These can be in nbl/hlsl/text_rendering/msdf.hlsl within correct namespace.
(Fix indentation please with the brackets) 

Do this when you've finalised working on these functions cause otherwise built-ins have to rebuild each time you make a change to the hlsl built-in.

Don't forget to document what the different input params and terms such as msdfPixelRange means and how it's related to the msdfgen lib (you can copy paste and quote from the lib above these functions)